### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jstraub**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,21 @@ This is an implementation of the CVPR '19 paper "DeepSDF: Learning Continuous Si
...
+@InProceedings{Park_2019_CVPR,
+author = {Park, Jeong Joon and Florence, Peter and Straub, Julian and Newcombe, Richard and Lovegrove, Steven},
+title = {DeepSDF: Learning Continuous Signed Distance Functions for Shape Representation},
+booktitle = {The IEEE Conference on Computer Vision and Pattern Recognition (CVPR)},
+month = {June},
+year = {2019}
+}
+```
+
+## Code Organization
```
Comment (tschmidt23) [inline README.md]: This section is more about organization on the filesystem than code organization... can we maybe change this to "Data Organization" or "File Organization"?

Comment (jstraub) [inline README.md]:
```suggestion
## File Organization
```

Comment (jstraub) [inline README.md]:
```suggestion
## File Organization
```

**Comment by tschmidt23** [comment]
Great, thanks Julian!

**Pull request merged**